src/dmd/root/serializer.d

```diff
@@ -0,0 +1,218 @@
+/**
+ * Compiler implementation of the
+ * $(LINK2 http://www.dlang.org, D programming language).
+ *
+ * Copyright:   Copyright (C) 1999-2019 by The D Language Foundation, All Rights Reserved
+ * Authors:     $(LINK2 http://www.digitalmars.com, Walter Bright)
```

Comment:
Just remove this for new files?

Reply:
Fine by me. I thought it needed to be consistent.

Reply:
Done.